pyproject.toml

```diff
@@ -39,25 +39,25 @@ keywords = [
     "ascii",
 ]
 dependencies = [
-    "numpy>=1.23",
+    "numpy>=1.23.2",
```

Comment:
Is the bugfix pin really necessary?

Reply:
1.23.0 and 1.23.1 didn't support Python 3.11